lib/ansible/module_utils/facts/hardware/openbsd.py

```diff
@@ -39,35 +39,28 @@ class OpenBSDHardware(Hardware):
     - processor_cores
     - processor_count
     - processor_speed
+    - uptime_seconds
 
     In addition, it also defines number of DMI facts and device facts.
     """
     platform = 'OpenBSD'
 
     def populate(self, collected_facts=None):
         hardware_facts = {}
-        self.sysctl = get_sysctl(self.module, ['hw', 'kern'])
+        self.sysctl = get_sysctl(self.module, ['hw'])
 
-        # TODO: change name
-        cpu_facts = self.get_processor_facts()
-        memory_facts = self.get_memory_facts()
-        device_facts = self.get_device_facts()
-        dmi_facts = self.get_dmi_facts()
-        uptime_facts = self.get_uptime_facts()
+        hardware_facts.update(self.get_processor_facts())
```

Comment:
i was going to make this a loop in which try/except would prevent issues with the underlying functions erroring out
```
try:
    hardware_facts.update(getattr(self, func)()) 
...
```

Reply:
Yeah, I think we should do something like this for all the hardware facts collectors. I'll write that in a separate PR.